webpack.config.js

```diff
@@ -92,15 +94,17 @@ module.exports = {
         compress: true,
         historyApiFallback: true,
         port: 8080,
-        server: "https",
-        proxy: [
-            {
-                // This proxies calls from the browser to the configured Foundry instance
-                target: process.env.API_PROXY_TARGET_URL,
-                context: ["/multipass/api/**", "/api/**"],
-                changeOrigin: true,
-                secure: true,
-            },
-        ],
+        server: useLocalhostInCORS ? "http" : "https",
+        proxy: useLocalhostInCORS
+            ? []
+            : [
+                  {
+                      // This proxies calls from the browser to the configured Foundry instance
+                      target: process.env.API_PROXY_TARGET_URL,
+                      context: ["/multipass/api/**", "/api/**"],
+                      changeOrigin: true,
+                      secure: true,
+                  },
+              ],
```

Comment:
i think i'd rather keep this clean and remove it, and move this into the FAQ in the README instead.